src/libcore/fmt/mod.rs

```diff
@@ -283,6 +289,7 @@ impl<'a> ArgumentV1<'a> {
 
     fn as_usize(&self) -> Option<usize> {
         if self.formatter as usize == ArgumentV1::show_usize as usize {
+            // SAFETY: if the formatter is `show_usize`, it means it came in as `&usize`
```

Comment:
This is actually not okay, see https://github.com/rust-lang/rust/issues/58320

In practice I guess it works out today, and is not necessary for safety in the sense that we're statically generating code that always has usize slots matching with show_usize anyway.

Reply:
Wait, was `Display` ever called `Show`? Why is it called `show_usize`?

Reply:
Yes, it used to be `trait Show` way back (killed when we "tossed tomatoes" in e3104d8f0c3ceb3b268cd6ddc5e91f355e44a7c5 :)

Reply:
That means it should be `display_usize` today.

Reply:
I added a FIXME for both issues.